Dockerfile

```diff
@@ -0,0 +1,41 @@
+# The MIT License
+#
+#  Copyright 2020 Loghi Perinpanayagam.
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to deal
+#  in the Software without restriction, including without limitation the rights
+#  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#  copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+
+
+FROM jenkins/ssh-agent
+
+# install python
+RUN apt-get update  && apt-get install -y \
+    python \
+    python-pip
+
+RUN apt install default-jdk -y
```

Comment:
This could go in the line above. But is that really necessary @Loghijiaha ?

I think the base image is an openjdk one, that comes with the openjdk already installed for Jenkins agents?

Looking at your image in Docker Hub, I can see at least in the path the openjdk folder... 

Or was there another reason for installing it?

https://hub.docker.com/layers/loghijiaha/ml-ssh-agent/latest/images/sha256-a5bafe67f3ad85593f81efbc77703fa9d42281932e448cc7cedb736afd81bafd?context=explore